app/src/main/res/layout/list_blood_sugar_history_item.xml

```diff
@@ -3,10 +3,7 @@
   xmlns:app="http://schemas.android.com/apk/res-auto"
   xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  android:background="?attr/selectableItemBackground"
```

Comment:
If this is removed, we won't have any touch feedback, right?

Reply:
It will be, but since there is no edit feature in this PR touch feedback doesn't seem to be needed. what do you think?

Reply:
Ah, that makes sense. We can add this when we implement the Edit feature.